valhalla/sif/dynamiccost.h

```diff
@@ -1017,7 +1017,7 @@ class DynamicCost {
   // Penalties that all costing methods support
   float maneuver_penalty_;         // Penalty (seconds) when inconsistent names
   float alley_penalty_;            // Penalty (seconds) to use a alley
-  float destination_only_penalty_; // Penalty (seconds) using private road, driveway, or parking aisle
+  float destination_only_penalty_; // Penalty (seconds) using destination only roads
```

Comment:
In fact the previous comment was more correct, however missing that it also includes actual „destination“ tags.

Reply:
True. I was completely focusing on destination only signs, but forgott about valhalla treating private roads (for good reasons) similar to destination only.